### PR TITLE
feature/dynamic-search-fields

### PR DESCRIPTION
**Pull request by mohamedAbouda**
add dynamic search fields

### CODE REVIEW

File: src/app/Http/Controllers/LaravelLoggerController.php [hunk 1]
```diff
@@ -63,21 +63,33 @@ private function mapAdditionalDetails($collectionItems)
      *
      * @return \Illuminate\Http\Response
      */
-    public function showAccessLog()
+    public function showAccessLog(Request $request)
     {
         if (config('LaravelLogger.loggerPaginationEnabled')) {
-            $activities = Activity::orderBy('created_at', 'desc')->paginate(config('LaravelLogger.loggerPaginationPerPage'));
+            $activities = Activity::orderBy('created_at', 'desc');
```
[the remark this answers is not in the document]
Comment (mohamedAbouda) [reply src/app/Http/Controllers/LaravelLoggerController.php]: the fix at line 73, what happened here first the descending order then searching if the search is enabled after that the pagination

File: README.md [hunk 2]
```diff
@@ -1,330 +1,28 @@
-# Laravel Activity Logger
+# Logger dynamic search
```
[the remark this answers is not in the document]
Comment (mohamedAbouda) [reply README.md]: i have returned the original readme and added "Search" section

[the remark this answers is not in the document]
Comment (mohamedAbouda) [reply README.md]: @jeremykenedy please, check again

File: src/app/Http/Controllers/LaravelLoggerController.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (mohamedAbouda) [reply src/app/Http/Controllers/LaravelLoggerController.php]: @jeremykenedy  any news ?

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (mohamedAbouda) [reply README.md]: @jeremykenedy ?????!

**Comment by jeremykenedy** [comment]
This is still on my radar. I have been busy with traveling, Laracon, and my other job :)

**Pull request merged**